src/ngx_http_lua_util.c

```diff
@@ -4327,5 +4364,36 @@ ngx_http_lua_escape_log(u_char *dst, u_char *src, size_t size)
     return 0;
 }
 
+ngx_int_t ngx_http_lua_copy_escape_header(ngx_http_request_t *r,
+    ngx_str_t *dst, ngx_int_t header_name)
```

Comment:
What does this `header_name` argument mean? If it's a boolean value, then we should use the type `unsigned` and parameter name `is_name` instead.